rr_endline_detection/src/endline_detection.cpp

```diff
@@ -0,0 +1,80 @@
+#include "endline_detection.hpp"
+
+//constructor
+EndlineCounter::EndlineCounter (ros::NodeHandle nh_) : it_(nh_) 
+{
+	//pub_ = nh_.advertise<std_msgs::Bool>("endline",1);
+	status = 0;
+	client_ = nh_.serviceClient<std_srvs::Trigger>("/Supervisor/count_lap", true);
+}
+
+//callback to handle detection
+void EndlineCounter::img_callback(const sensor_msgs::ImageConstPtr& msg)
```

Comment:
function naming should be consistent and follow https://google.github.io/styleguide/cppguide.html#Naming

Reply:
done by you